python/orca/src/bigdl/orca/learn/pytorch/utils.py

```diff
@@ -353,6 +353,22 @@ def index_concatenate(x, axis=0):
         return res
 
 
+def split_predict_cols(y):
+    if not isinstance(y, (list, tuple)):
+        return {"prediction": y}
+    output_cols = [f"output_{i}" for i in range(len(y))]
+    if isinstance(y, np.ndarray):
+        # Single output
+        y = [y]
```

Comment:
the logic here seems strange. For the single output case, it is already considered in Line358 right?

Reply:
Yes, this is not working and should be aborted.